data/json/recipes/recipe_deconstruction.json

```diff
@@ -4907,6 +4907,15 @@
     "components": [ [ [ "felt_patch", 10 ] ] ],
     "flags": [ "BLIND_EASY" ]
   },
+  {
+    "result": "bundle_pipe",
+    "type": "uncraft",
```

Comment:
I don't think you need the uncraft if you made the recipe reversible and this uncraft will destroy whatever rope you used to craft it while reversible recipes should uncraft back into their used components whatever they were.

Reply:
I have deleted the uncraft recipe.